app/services/cycle_timetable.rb

```diff
@@ -0,0 +1,100 @@
+class CycleTimetable
+  CYCLE_DATES = {
+    2021 => {
+      find_opens: DateTime.new(2020, 10, 6, 9, 0, 0),
+      apply_opens: DateTime.new(2020, 10, 13, 9, 0, 0),
+      deadline_banner: DateTime.new(2021, 7, 12, 9, 0, 0),
```

Comment:
Wonder if for consistency we should use event-y type language.
e.g. 'course_search_deadline` - or something. TBH I'm not sure which deadline it refers to ha ha

Reply:
Yea good point. It's basically just when the deadline banners should start rendering so i didn't have to do something like. apply_1_deadline - 6.weeks or w/e the calculation would be for it starting to show on Monday.